### ISSUE TITLE
404 pages broken

### ISSUE THREAD

**Issue by nmalcolm** [mybb/mybb.com#12]
I like the irony. The 404 page cannot be found. :)

http://www.mybb.com/foobar

**Comment by pkill37** [comment]
No idea why it's not working.

The docs' 404 page isn't working either.

Any ideas?

**Comment by nmalcolm** [comment]
It might be because the branch is "gh-pages" and not "master". IIRC master needs to be used for org. sites. I might be talking complete nonsense, though.

**Comment by JordanMussi** [comment]
The 404 pages should work... If using mybb.github.io/... to publish pages then only the mybb.github.io 404 file will be used (this repo will be use master). When publishing to your own website the 404 for each repo should work (these will use gh-pages).

Any who, I'm not sure what to suggest to fix it. :wink:

**Comment by JordanMussi** [comment]
They should work as we're using a custom domain...
https://help.github.com/articles/custom-404-pages

**Comment by Kaeden** [comment]
Solved in 653751717e

**Comment by davidsilvasmith** [comment]
It seems to be broken again? http://www.mybb.com/asdf

I have the same issue :(

**Comment by Destroy666x** [comment]
@davidsilvasmith , I can see a custom 404 page

**Comment by davidsilvasmith** [comment]
Yes you are right. Looks like it was a Github issue that they fixed. Mine is working now too. Looks like the custom 404s might just break. I noticed you guys added the permalink, which I did also, but I have [another site without the permalink](https://raw.githubusercontent.com/davidsilvasmith/sowhatsbitcoin.com/gh-pages/404.md): /404.html and that was broken yesterday but now it is working fine.

So whatever fixed at Github fixed my site without the permalink: /404.html 404 page.
